src/audio.c

```diff
@@ -1044,33 +1044,65 @@ static int set_ebuacip_params(struct audio *au, uint32_t ptime)
 		if (0 == str_cmp(str, "auto")) {
 
 			err |= sdp_media_set_lattr(sdp, false,
-						   "ebuacip",
-						   "jbdef %i auto %d-%d",
-						   jb_id,
-						   avt->jbuf_del.min * ptime,
-						   avt->jbuf_del.max * ptime);
+					"ebuacip",
+					"jbdef %i auto %d-%d",
+					jb_id,
+					avt->jbuf_del.min * ptime,
+					avt->jbuf_del.max * ptime);
 		}
 		else if (0 == str_cmp(str, "fixed")) {
 
 			/* define jb value in option */
 			jbvalue = avt->jbuf_del.max * ptime;
 
 			err |= sdp_media_set_lattr(sdp, false,
-						   "ebuacip",
-						   "jbdef %i fixed %d",
-						   jb_id, jbvalue);
+							"ebuacip",
+							"jbdef %i fixed %d",
+							jb_id, jbvalue);
 		}
 	}
 
 	/* set QOS recomendation use tos / 4 to set DSCP value */
 	err |= sdp_media_set_lattr(sdp, false, "ebuacip", "qosrec %u",
-				   avt->rtp_tos / 4);
+							   avt->rtp_tos / 4);
 
 	/* EBU ACIP FEC:: NOT SET IN BARESIP */
 
 	return err;
 }
 
+static bool ebuacip_handler(const char *name, const char *value, void *arg)
+{
+	struct sdp_media *sdp;
+	struct audio *au = arg;
+	struct aurx *rx = &au->rx;
+	struct pl type, val;
+	uint32_t frames;
+
+	if (0 == re_regex(value, str_len(value),
+		"jbdef [0-9]+ [^ ]+ [0-9]+",
+		NULL, &type, &val))	{
+
+		frames = pl_u32(&val) / rx->ptime;
```

Comment:
we should check if rx->ptime is zero, to avoid potential division by zero errors.